src/plugin.css

```diff
@@ -0,0 +1,30 @@
+.vjs-contextmenu-ui-menu {
+  position: absolute;
+}
+
+.vjs-contextmenu-ui-menu .vjs-menu-content {
```

Comment:
I could not get nesting to work correctly without cryptic errors from csso. Because this is such a small CSS file, I didn't feel like it was worth digging into it.